pkg/bicep/bicep.go

```diff
@@ -0,0 +1,41 @@
+package bicep
+
+import (
+	"encoding/json"
+	"fmt"
+	"path"
+
+	"github.com/massdriver-cloud/mass/pkg/bundle"
+	"github.com/massdriver-cloud/mass/pkg/inputvars"
+	"github.com/spf13/afero"
+)
+
+const paramsFile = "template.parameters.json"
+
+var template = map[string]interface{}{
```

Comment:
This works, but it would probably be better as a struct, that is instantiated and used in GenerateFiles, instead of a package level map. It would rely more heavily on Go's type-safety and provide more control of JSON serialization, if needed.